SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/DispatcherTest.cpp

```diff
@@ -45,32 +31,32 @@ DISPATCHER_TEST(GIVEN_Dispatcher_WHEN_Callback_Added_Then_Invoked_THEN_Callback_
 
 	const SpatialGDK::FComponentValueCallback Callback = [&Invoked](const SpatialGDK::FEntityComponentChange& Change) {
 		if (Change.EntityId == ENTITY_ID && Change.Change.ComponentId == COMPONENT_ID
-			&& GetValueFromSchemaComponentData(Change.Change.Data) == COMPONENT_VALUE)
+			&& SpatialGDK::GetValueFromSchemaComponentData(Change.Change.Data) == COMPONENT_VALUE)
 		{
 			Invoked = true;
 		}
 	};
 	Dispatcher.RegisterComponentAddedCallback(COMPONENT_ID, Callback);
 
 	AddEntityToView(View, ENTITY_ID);
-	PopulateViewDeltaWithComponentAdded(Delta, View, ENTITY_ID, COMPONENT_ID, COMPONENT_VALUE);
+	PopulateViewDeltaWithComponentAddedWithValue(Delta, View, ENTITY_ID, COMPONENT_ID, COMPONENT_VALUE);
```

Comment:
I think I made this comment on another PR, but this seems like it would be clearer and more reusable if it didn't take a double and just took a ComponentData. Then you call it with a test component.
Especially as all the other functions we have that do similar things do that.

Reply:
I think you're totally right, and blindly copied that we were normally using doubles as the data inside test component data. Will change this

Reply:
done